src/main/java/step2/domain/lotto/LottoNumber.java

```diff
@@ -0,0 +1,18 @@
+package step2.domain.lotto;
+
+import static step2.LottoNumberGenerator.*;
+
+public class LottoNumber {
+    private Integer number;
```

Comment:
이 값도 한번 정해지면 바뀌지 않을 것 같은데 아래와 같이 선언하면 어떨까요?

```suggestion
    private final Integer number;
```